lib/resque/worker.rb

```diff
@@ -503,17 +503,15 @@ def heartbeat!(time = Time.now)
     end
 
     def start_heartbeat
-      heartbeat!
```

Comment:
Is there a specific reason this was being done outside the thread to begin with?